payments/schema/mutations.py

```diff
@@ -394,52 +394,67 @@ def mutate_and_get_payload(cls, root, info, **input):
         return CreateAdditionalProductOrderMutation(order=order)
 
 
+class UpdateOrderInput(OrderInput, graphene.InputObjectType):
+    id = graphene.ID(required=True)
+
+
 class UpdateOrderMutation(graphene.ClientIDMutation):
```

Comment:
Should this be renamed to `UpdateOrdersMutation`?

Or can we even keep them as separate mutations?

Reply:
hmm yes UpdateOrder`s`Mutation makes more sense

Reply:
UpdateOrdersMutation has all functionality that the old UpdateOrderMutation had, so I don't think it's worth keeping

Reply:
Fair enough, I guess if we need the individual one, we can add it later.